garnish/parse.py

```diff
@@ -0,0 +1,266 @@
+# Copyright 2019-2019 the garnish authors. See copying.md for legal info.
+
+import re
+import sys
+import os
+import subprocess
+from pymol import cmd
+
+# local imports
+from .utils import get_gmx, update_recursive, clean_path
+
+
+def parse_top(top_file):
```

Comment:
If we have a single parse.py file, we might as well remove the "parse_" in front of the function names.
So we can have parse.top and parse.tpr instead of parse.parse_top etc

Reply:
good point.